uplinks/freifunk-berlin-uplink-tunnelberlin-tunneldigger-files/uci-defaults/freifunk-berlin-z95_tunnelberlin-tunneldigger

```diff
@@ -27,17 +24,9 @@ uci set network.ffuplink_dev.name=ffuplink
 # Create a static random macaddr for ffuplink device
 macaddr=$(macaddr_canonicalize `dd 2>/dev/null bs=1 count=6 </dev/urandom | hexdump -e '1/1 "%02x"'`)
 uci set network.ffuplink_dev.macaddr=$(macaddr_setbit_la $macaddr)
+uci commit network.ffuplink_dev
 
-uci delete network.ffuplink
-uci set network.ffuplink=interface
-uci set network.ffuplink.ifname=ffuplink
-uci set network.ffuplink.proto=dhcp
-# Put the resulting routing information from the dhcp request in the ffuplink table
-# instead of the default table.  This prevents the dhcp request for the ffuplink
-# interface from overwriting the routing table entries needed by br_wan.
-uci set network.ffuplink.ip4table=ffuplink
-uci set network.ffuplink.ip6table=ffuplink
-uci commit network
+create_ffuplink
```

Comment:
proto needs to be set to dhcp after calling create_ffuplink, followed by another uci commit network